src/Options/index.js

```diff
@@ -94,6 +94,12 @@ export interface ParseServerOptions {
   databaseOptions: ?DatabaseOptions;
   /* Adapter module for the database */
   databaseAdapter: ?Adapter<StorageAdapter>;
+  /* Disable case insensitivity (collation) on queries and indexes, needed if the you use mongodb serverless
+  :ENV: PARSE_SERVER_DISABLE_DEFAULT_CASE_INSENSITIVITY */
```

Comment:
- Is it necessary to specify the ENV parameter? The options script should create that automatically from the param name. That also prevents the mistake, that the env var name is different than the param name, as is here.
- Please use correct spelling ("MongoDB") and notation, this should be spelled like full sentences, as it goes into the API docs as prose text. 
- Can we find a param name that is easier to read and more descriptive, for example avoiding the double negation, for example `enable...` with a default value of `true`?